LayoutTests/http/tests/workers/service/shownotification-invalid-data-expected.txt

```diff
@@ -0,0 +1,8 @@
+This tests that a notification shown by a service worker fails if it is passed an invalid data option
+
+On success, you will see a series of "PASS" messages, followed by "TEST COMPLETE".
```

Comment:
We don't see "TEST COMPLETE" at the end of your test so you're probably doing something wrong.